modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -1,173 +1,30 @@
-= Create and Configure Message Types
+= Creating Message Types
 
-Message types define the structure of the transactions that Anypoint Partner Manager receives or sends. You can create message types from the *<host>* page or the Message Flow wizard.
+You can create new message types from the *<host>* page or by using the Message Flow wizard.
 
-[[create-message-type]]
```

Comment:
If you remove an anchor, please ensure that it is not being called from any file. 
Even if it is not used, you can leave it in the article as it does not disrupt content in any way.